docs/reference/supported-environments.md

```diff
@@ -50,35 +68,58 @@ Requirements for the components can be seen below:
 Camunda 8 works with the [default distribution](https://www.elastic.co/downloads/elasticsearch) of Elasticsearch, which is available under the [Free or Gold+ Elastic license](https://www.elastic.co/pricing/faq/licensing#summary).
 :::
 
+### Volume Performance
+
+Depending on your cloud provider, we offer the following volume performance recommendations:
+
+- [Amazon EKS](/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks#volume-performance)
+- [Google GKE](/self-managed/platform-deployment/helm-kubernetes/platforms/google-gke#volume-performance)
+- [Microsoft AKS](/self-managed/platform-deployment/helm-kubernetes/platforms/microsoft-aks#volume-performance)
+- [Red Hat OpenShift](#) <!-- No current documentation available. -->
+
 ### Helm chart version matrix
 
 The core Camunda components have a unified fixed release schedule following the [release policy](./release-policy.md). However, some of the applications have their own schedule. The following compatibility matrix gives an overview of the different versions with respect to the Helm chart versions.
 
 Since the 8.4 release (January 2024), the Camunda Helm chart version is decoupled from the version of the application (e.g., the chart version is 9.0.0 and the application version is 8.4.x).
 
+<!-- Nit pick - this is a list, not a matrix. This is also on a separate site with no oversight from DevEx. -->
```

Comment:
@MaxTru we should have a discussion about either collapsing these into docs.camunda.io or getting DevEx involved in reviews for helm.camunda.io.